src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -95,6 +95,11 @@ private static void OnProcessExit(object sender, EventArgs e)
         {
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
 
+            if(_logger.PiiLoggingEnabled)
```

Comment:
I think `if` needs to be removed and we need to call `EnablePii` every time RuntimeBroker is created.

The scenario I'm thinking:
- PCA instance is created with PII logging.
- ATI call is made
- Interop instance is created statically.
- RuntimeBroker instance is created, PII enabled in Interop static instance.
- For some other ATI call, a new PCA instance is created. PII logging is disabled (dynamically somehow, through config or whatever).
- Interop instance is already created.
- RuntimeBroker instance is created, **PII needs to be disabled here** for the static interop instance.

Reply:
Good point. The irony here is that there is only one static instance of the interop and there can be multiple instances of runtime broker. Let's take the above condition a step further. 
User starts utilizing the first instance again. In that case, Pii will not be enabled as the constructor will not be called.

In most cases Pii is enabled/disabled for the lifespan of the application. So, it is unlikely that the case you mentioned will ever happen.
The reason if was put was there to avoid unnecessary interop call that may impact perf.

Reply:
> User starts utilizing the first instance again. In that case, Pii will not be enabled as the constructor will not be called.

Yes, that's right. I think that scenario of having multiple PCA instances that are reused is uncommon. The more common scenarios are either using a static PCA instance or recreate a new instance for each request.

> In most cases Pii is enabled/disabled for the lifespan of the application. So, it is unlikely that the case you mentioned will ever happen.

Yea, I think for PCA that's common, since PCA apps are closed/opened all the time, as opposed to CCA running on a server. The scenario I was thinking of is if I'm using an app and something breaks, so I enable logs in the settings, then try to repro again - this would mean I'd have to restart the app (so that interop instance gets recreated with new settings).

Probably unlikely I guess, but wanted to bring it up so it doesn't create weird issues down the road.